README.md

```diff
@@ -177,11 +177,12 @@ Supported Features
 ------------------
```

Comment:
@abhi-zapr @gagangupt16 Since project is re-released under Apache 2.0 we will need new license text